Gemfile.lock

```diff
@@ -209,60 +209,60 @@ GEM
       sprockets-rails
       tilt
     semantic_range (2.3.0)
-    spring (2.1.0)
+    spring (2.1.1)
     spring-watcher-listen (2.0.1)
       listen (>= 2.7, < 4.0)
       spring (>= 1.2, < 3.0)
     sprockets (4.0.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
-    sprockets-rails (3.2.1)
+    sprockets-rails (3.2.2)
       actionpack (>= 4.0)
       activesupport (>= 4.0)
       sprockets (>= 3.0.0)
     thor (1.0.1)
     thread_safe (0.3.6)
     tilt (2.0.10)
-    tinymce-rails (5.4.1)
+    tinymce-rails (5.6.1)
       railties (>= 3.1.1)
-    trestle (0.9.3)
+    trestle (0.9.4)
       activemodel (>= 4.2.0)
       kaminari (>= 1.1.0)
       railties (>= 4.2.0)
       sprockets-rails (>= 2.0.0)
-    trestle-active_storage (3.0.0)
+    trestle-active_storage (3.0.1)
       rails (>= 5.2, < 7)
       trestle (~> 0.9.0, >= 0.9.3)
-    trestle-auth (0.4.0)
+    trestle-auth (0.4.1)
```

Comment:
### [trestle-auth](https://www.trestle.io)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.4.0 → 0.4.1

[_~~change-log~~, [source-code](https://github.com/TrestleAdmin/trestle-auth)_]





<details>
<summary>Commits</summary>

A change of **10** commits. See the full changes on [the compare page](https://github.com/TrestleAdmin/trestle-auth/compare/v0.4.0...v0.4.1).

These are the individual commits:
- (5f3e75c) [Admin generator: don't re-login after an invalid password ch…](https://github.com/TrestleAdmin/trestle-auth/commit/5f3e75cc85bfc4c449e322618bec36e71e6c93bb)
- (f318a67) [Clean up admin generator](https://github.com/TrestleAdmin/trestle-auth/commit/f318a67eb8df4d077e143b06640301850938bc79)
- (7ad9ca6) [Implement account admin generator](https://github.com/TrestleAdmin/trestle-auth/commit/7ad9ca6d5575fad650933d40d34c08c34ce4c377)
- (fd69e23) [Pass local variables f in login form params](https://github.com/TrestleAdmin/trestle-auth/commit/fd69e239efa8d3b1186401ff86f7d4dc605308cb)
- (96753cc) [Merge pull request #29 from McRipper/master](https://github.com/TrestleAdmin/trestle-auth/commit/96753cc902fb7fad21a3356eddeba338707bd356)
- (156576a) [Update trestle dependency to >= 0.9.3](https://github.com/TrestleAdmin/trestle-auth/commit/156576ad93809f1b1a829cbd70fb3610c96292cd)
- (9a5b258) [cuenta instead of conta](https://github.com/TrestleAdmin/trestle-auth/commit/9a5b25870724229da93362d50da16ad88eb1be3f)
- (1f94490) [Merge pull request #30 from cristianvirtual/es_translation_f…](https://github.com/TrestleAdmin/trestle-auth/commit/1f94490aadaa94f104b65ba1fdda48a2def4f255)
- (9860708) [Update JS pipeline dependencies](https://github.com/TrestleAdmin/trestle-auth/commit/9860708ad16568d0340349d0402107894484a0a4)
- (71d7f3d) [Increment version to 0.4.1](https://github.com/TrestleAdmin/trestle-auth/commit/71d7f3d9f32d2f2cae385a9d9beb397a142b9203)


</details>